src/assignment/queries/get-assignment-by-uuid.graphql

```diff
@@ -1,5 +1,5 @@
 query getAssignmentByUuid($id: uuid!) {
-	app_assignments_v2_overview(where: { id: { _eq: $id }, is_deleted: { _eq: false } }) {
+	app_assignments_v2(where: { id: { _eq: $id }, is_deleted: { _eq: false } }) {
```

Comment:
I thought this table was deprecated and we need to use the `_overview` table?
@bertyhell